uniflow/flow/transform/transform_news_feed_flow.py

```
@@ -0,0 +1,246 @@
+            prompt_template (PromptTemplate): Guided prompt template.
+            model_config (Dict[str, Any]): Model config.
+        """
+        # TODO: Refactoring needed to make model_op output Context format. Need to keep it in Context format and only convert back to dictionary format before exiting Flow
+        super().__init__()
+        if model_config["response_format"]["type"] == "json_object":
+            model = JsonLmModel(
+                prompt_template=prompt_template,
+                model_config=model_config,
+            )
+        else:
+            model = LmModel(
```

Ruff (0.8.2)

Local variable `model` is assigned to but never used

Remove assignment to unused variable `model`

(F841)